arviz/tests/external_tests/test_data_pymc.py

```diff
@@ -139,6 +139,16 @@ def test_from_pymc_predictions(self, data, eight_schools_params):
             assert ivalues.shape[0] == 1  # one chain in predictions
             assert np.all(np.isclose(ivalues[0], values))
 
+    def test_from_pymc_trace_inference_data(self):
+        # check if error is raised successfully after passing InferenceData as trace
+        with pm.Model():
+            p = pm.Uniform("p", 0, 1)
+            pm.Binomial("w", p=p, n=2, observed=1)
+            trace = pm.sample(100, chains=2, return_inferencedata=True)
+            assert isinstance(trace, InferenceData)
```

Comment:
I think we don't need to sample from a model, we can use an example inferencedata or create it with `from_dict` and then do `az.from_pymc3(idata, model=pm.Model())` directly